signer/src/config/mod.rs

```diff
@@ -23,6 +24,8 @@ use crate::stacks::wallet::SignerWallet;
 mod error;
 mod serialization;
 
+pub const MAX_BITCOIN_PROCESSING_DELAY_S: u64 = 300;
```

Comment:
It looks like the build is failing because this constant is missing docs (we have a lint to make sure we have documentation for everything. Would you mind addressing this in another PR? Also, would you mind renaming this to `MAX_BITCOIN_PROCESSING_DELAY_SECS` or something similar?

Reply:
Ops, sorry! I have seen failing tests locally and attributed the CI failure to that. Let me fix that right away.